### ISSUE TITLE
Gesture not working

### ISSUE THREAD

**Issue by GiovanniL19** [html-next/ember-gestures#63]
Hi,

I have setup a swipe recogniser under ember-gestures > recognizers >swipe.js. The code is as follows:

```
export default {
  include: [],
  exclude: [],
  options: { threshold: 25, direction: typeof Hammer === 'undefined' ? '' : Hammer.DIRECTION_HORIZONTAL },
  recognizer: 'swipe'
};
```

On a div element I have the following action: 

`{{action "remove" object on="swipe"}}`

Does anyone know if I am missing something? The gestures are not working with this setup.

Thank you in advance

**Comment by eriktrom** [comment]
@GiovanniL19 - try mixing in the `RecognizerMixin` to your component (like here: https://github.com/runspired/ember-gestures/blob/31ce7c74bff5eadbfdfa7bcf4c4539ae85e310fa/addon/components/slide-toggle.js#L18) and adding the `recognizers` property like here: https://github.com/runspired/ember-gestures/blob/31ce7c74bff5eadbfdfa7bcf4c4539ae85e310fa/addon/components/slide-toggle.js#L24

tl;dr - 

1. Add the `RecognizerMixin` to your component [like here:](https://github.com/runspired/ember-gestures/blob/31ce7c74bff5eadbfdfa7bcf4c4539ae85e310fa/addon/components/slide-toggle.js#L18) 
2. then add a property to your component, [like here:](https://github.com/runspired/ember-gestures/blob/31ce7c74bff5eadbfdfa7bcf4c4539ae85e310fa/addon/components/slide-toggle.js#L24): 
```js
recognizers: 'swipe'
```

**Comment by eriktrom** [comment]
closing - re-open if u seek further help
